src/picongpu/include/particles/ionization/None/None.def

```diff
@@ -27,9 +29,33 @@ namespace particles
 namespace ionization
 {
 
+    /** \struct None_Impl
```

Comment:
I'll put in a new line after the headline but I'd like to keep the `\brief` because in some docs I also write sth longer later without a `\details` command.